### DISCUSSION TITLE
Differentiate between faction generators

### DISCUSSION THREAD

**Discussion by SiegeSpud** [OpenHV/OpenHV#884] [Ideas]
To further emphasize faction differences generators should not be just reskins of the same building. YI as the more mobile faction could have the current generator, but Synapol's could be more expensive and provide more power with less efficiency as in the example below.

- YI 300$ = 120 Power Units (1PU = 2.50$)
- SY 400$ = 150 Power Units (1PU = 2.67$)

**Comment by OcelotWalrus** [comment]
It seems to be a nice balance. Maybe the new look of the synapol generator would be bigger then? Like 3x2 tiles instead of 2x2.

**Comment by WilliamWeberBerrutti** [comment]
I think we discussed it on Matrix channel, but overall this shouldn't be good for the game, balance-wise. It'd be hard to properly balance this idea with low gains for the players. The Generators seem fine the way they are right now.
